### ISSUE TITLE
Take down readthedocs site

### ISSUE THREAD

**Issue by pwildenhain** [redcap-tools/PyCap#199]
New site is here: redcap-tools.github.io/pycap/

don't want people to get confused.

I think only @sburns has permission to do this

**Comment by sburns** [comment]
This URL 404s for me.

**Comment by pwildenhain** [comment]
This one? https://redcap-tools.github.io/PyCap/

**Comment by remifrazierucsf** [comment]
@sburns +1 for this.

**Comment by remifrazierucsf** [comment]
@pwildenhain your original comment in this issue has `pycap` instead of `PyCap` in the URL, and the github.io server enforces case sensitivity.

**Comment by pwildenhain** [comment]
SMH @remifrazierucsf you're right 🤦🏻‍♂️ 

Here is the _actual_ link: http://redcap-tools.github.io/PyCap/ once again -- I got messed up because the other on the sidebar is all lowercase, and I just copied and pasted that into this issue rather than copying and pasting the true url from the site itself

@remifrazierucsf Any thoughts about the new docs while you're here?

@sburns Are you able to see the updated link?

**Comment by sburns** [comment]
Yes this link works fine, looks great 👍🏻 . What is necessary from my side? Just deleting the read the docs site?

**Comment by pwildenhain** [comment]
That would be great, thank you!

**Comment by pwildenhain** [comment]
Thanks Scott!
